addons/io_scene_gltf2/__init__.py

```diff
@@ -1756,7 +1927,10 @@ def menu_func_import(self, context):
 
 def register():
     from .blender.com import gltf2_blender_ui as blender_ui
+
     for c in classes:
+        if (hasattr(c, "bl_label") is True and c.bl_label == GLTF_PT_export_gltfpack.bl_label):
```

Comment:
I think we should always register the class, and add a check in poll of UI to avoid display if not needed. This will avoid this check

Reply:
Makes sense.